### PR TITLE
[Plugin] Invalidate window after changing its colours

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
@Manticore-007, this one's for you

### CODE REVIEW

**Comment by Manticore-007** [comment]
You're the best!

**Review by Gymnasiast — COMMENTED** [review]
Worth a changelog entry, you reckon?

**Review by Manticore-007 — APPROVED** [review]
I gave it a test run, and now the window and its widgets change colour as it should.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - Fix: [#23023] Large scenery clearance height interpreted as negative when greater than 127.
 - Fix: [#23044] "remove_unused_objects" command causes blank peep names.
 - Fix: [#23048] Map generator allows map sizes out of range through text input.
+- Fix: [#23058] [Plugin] Changing window colours doesn't trigger the window to be fully redrawn.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:16]:
```suggestion
- Fix: [#23058] [Plugin] Changing window colours doesn’t trigger the window to be fully redrawn.
```

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**